Cargo.toml

```diff
@@ -16,24 +16,27 @@ members = [".", "tendermint-rs"]
 [badges]
 circle-ci = { repository = "tendermint/kms" }
 
+[dependencies.abscissa]
+version = "0.1.0-pre.1"
+default-features = false
+features = ["application", "signals", "secrets", "time"]
+
 [dependencies]
-abscissa = "0.0.6"
-abscissa_derive = "0.0.2"
 atomicwrites = "0.2"
 byteorder = "1.2"
 bytes = "0.4"
 chrono = "0.4"
 failure = "0.1"
-failure_derive = "0.1"
+gumdrop = "0.5"
```

Comment:
This is only used indirectly (via [derive(Options)], right? (I could not find any sign of `#[macro_use] extern crate gumdrop;`)

Reply:
I've largely switched to using Rust 2018 conventions for importing macros, where instead of `#[macro_use]` spooky-action-at-a-distance, macros can be imported by name just like anything else.

This allows a crate like `gumdrop` to re-export the proc macros from `gumdrop_derive` so that consumers don't need both. After the 2018 edition was released, pretty much everything with a proc macro crate switched to this convention (including `abscissa`).